src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/Apprentice/Details.cshtml

```diff
@@ -59,64 +55,64 @@
             <th scope="row" class="govuk-table__header das-table-cell-width-50">Employer</th>
             <td class="govuk-table__cell das-table-cell-width-50">@Model.Employer</td>
         </tr>
-    <tr class="govuk-table__row">
-        <th scope="row" class="govuk-table__header">Reference</th>
-        <td class="govuk-table__cell">@Model.Reference</td>
-    </tr>
-    <tr class="govuk-table__row">
-        <th scope="row" class="govuk-table__header">Agreement ID</th>
-        <td class="govuk-table__cell">@Model.AgreementId</td>
-    </tr>
-
+        <tr class="govuk-table__row">
+            <th scope="row" class="govuk-table__header">Reference</th>
+            <td class="govuk-table__cell">@Model.Reference</td>
+        </tr>
+        <tr class="govuk-table__row">
+            <th scope="row" class="govuk-table__header">Agreement ID</th>
+            <td class="govuk-table__cell">@Model.AgreementId</td>
+        </tr>
     </tbody>
 </table>
 
-
-
 <div class="govuk-grid-row govuk-!-margin-bottom-2">
     <div class="govuk-grid-column-two-thirds">
         <h2 class="govuk-heading-s govuk-!-margin-bottom-1">Apprentice</h2>
     </div>
     <div class="govuk-grid-column-one-third das-!-text-align-right--from-tablet">
-        @*<a href="#" class="govuk-link">Edit apprentice</a>*@
+        @if (Model.AllowEditApprentice)
+        {
+            <a href="@Model.EditApprenticeURL" class="govuk-link">Edit apprentice</a>
```

Comment:
Could we pop a sensible `Id` attribute onto this anchor, so that our testing friends can select it easily enough in automation?